src/browser/modules/D3Visualization/lib/visualization/utils/arrays.ts

```diff
@@ -27,7 +27,7 @@ export default function cloneArray<T>(original: Array<T>): Array<T> {
   return clone
```

Comment:
I'm not sure if the `clone` method makes any sense> It's not even a deep copy of each element. And I checked the usage, it never manipulates the indexes. Can you get rid of it?